### PR TITLE
Skipping 0 days 0 hours in gotgud

### PR DESCRIPTION
**Pull request by MetalBall887**

### CODE REVIEW

**Comment by algmyr** [comment]
Thanks for the commit. However please change this to use 4 spaces for indentation to match the rest of the project.

**Comment by MetalBall887** [comment]
Please check if it has the required indentation

File: tle/cogs/codeforces.py [hunk 1]
```diff
@@ -242,7 +242,8 @@ def check(problem):
             return
 
         def pretty_time_delta(seconds):
-            seconds = int(seconds)
+            
+            seconds = int (input ())
```
Comment (algmyr) [inline tle/cogs/codeforces.py]: Surely this is supposed to be `int(seconds)`. I presume you were testing things locally.

File: tle/cogs/codeforces.py [hunk 2]
```diff
@@ -252,8 +253,17 @@ def pretty_time_delta(seconds):
                 (hours, 'hour', 'hours'),
                 (minutes, 'minute', 'minutes'),
             ]
+
+            timeprint = []
+            for count, singular, plural in timespec:
+                if count:
+                    timeprint.append ((count, singular, plural))
```
Comment (algmyr) [inline tle/cogs/codeforces.py]:
```
timeprint = [(count,singular,plural) for count,singular,plural in timespec if count]
```

File: tle/cogs/codeforces.py [hunk 3]
```diff
@@ -252,8 +253,17 @@ def pretty_time_delta(seconds):
...
+
+            if not timeprint:
+                timeprint.append ((seconds, 'second', 'seconds'))
```
Comment (algmyr) [inline tle/cogs/codeforces.py]: Remove the space between function name and parenthesis.

**Pull request merged**